src/test/java/validationsOfManageUsersPage/ManageUserPageBaseTest.java

```diff
@@ -0,0 +1,54 @@
+package validationsOfManageUsersPage;
+
+import java.io.IOException;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.ITestResult;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Parameters;
+
+import pages7daySuperMarket.DashBoardPage;
+import pages7daySuperMarket.LoginPage;
+import pages7daySuperMarket.ManageOrdersPage;
+import pages7daySuperMarket.ManageUsersPage;
+import utils.DriverFactory;
+import utils.GetWaitMethod;
+import utils.ScreenShotUtility;
+
+public class ManageUserPageBaseTest {
+	WebDriver driver;
+	@BeforeClass
+	@Parameters({"browserName"})
+	public void initializeDriver(String browser) {
+		driver=DriverFactory.driverInitialization(browser);
+	}
+		@BeforeMethod
+		public void navigateToManageOrderPage() {
+			DriverFactory.loginToGroceryApp(driver);
+			
+			ManageUsersPage manageUserspage = new ManageUsersPage(driver);
+			manageUserspage.clickOnManageUsersButton();
+		}
+
+		@AfterMethod
+		public void screenShot(ITestResult result) throws IOException {
+			if (result.getStatus() == ITestResult.SUCCESS) {
```

Comment:
39-43 re[eating in all basetesters - consider moving this to DriverFactory classs